spec/std/file_spec.cr

```diff
@@ -27,11 +27,23 @@ private def it_raises_on_null_byte(operation, &block)
   end
 end
 
+private def normalize_permissions(permissions, *, directory)
```

Comment:
Windows doesn't have the concept of "execute permissions" (at least not at this level of the API)

Reply:
Yeah I don't expect to do the full ACL.

But either the feature is not enabled/tested on windows, or
there is some translation in the implementation to map (at some extent) the result from windows to the stdlib permission value and have a unified API in both platform.

Neither of these options requires a `normalize_permission` in the specs.

Reply:
It maps incoming permissions just fine, the problem is that it's a lossy transform, so that when the spec suite comes back and queries the permissions it just set on file xyz it gets different permissions to what it just set back.